epinar/assignment5/unit_tests

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import unittest
+from unknown_alphabet import find_alphabets, find_alphabet
+
+class TestDictionary(unittest.TestCase):
+
+	def check_equal(self, L1, L2):
+		return sorted(L1) == sorted(L2)
+
+	def test_find_alphabets(self):
+		self.check_equal(find_alphabets(['ART', 'RAT', 'CAT']), [['T', 'A', 'R', 'C'], ['A', 'T', 'R', 'C'], ['A', 'R', 'T', 'C'], ['A', 'R', 'C', 'T']])
+		self.check_equal(find_alphabets(['ART', 'RAT', 'CAT', 'CAR']), [['T', 'A', 'R', 'C'], ['A', 'T', 'R', 'C']])
+		self.check_equal(find_alphabets(['S', 'ART', 'RAT', 'CAT', 'CAR']), [['S', 'A', 'T', 'R', 'C'], ['S', 'T', 'A', 'R', 'C'], ['T', 'S', 'A', 'R', 'C']])
+
+	def test_find_alphabet(self):
+		self.assertEqual(find_alphabet(["1a", "1b", "a1", "ab"]), ["1", "a", "b"])
+
+
```

Comment:
I modified my own tests a bit and added some new for testing your solution. Your solution passed these tests.
You can add them if you want:
```
    def test_0(self):
        self.assertEqual(find_alphabet([]), [])
        self.assertEqual(find_alphabets([]), [[]])

    def test_1(self):
        self.assertEqual(find_alphabets(["a"]), [["a"]])
        self.assertEqual(find_alphabet(["1", "2", "3"]), ["1", "2", "3"])
        self.check_equal(find_alphabets(["12"]), [["1", "2"], ["2", "1"]])

    def test_2(self):
        self.assertEqual(find_alphabets(["1a", "1b", "a1", "ab"]), [["1", "a", "b"]])
        self.check_equal(find_alphabets(["1b", "a1"]), [["1", "a", "b"], ["1", "b", "a"]])

    def test_fault(self):
        self.assertEqual(find_alphabets(["1", "2", "1"]), [[]])
        self.assertEqual(find_alphabets(["12", "22", "21"]), [[]])
```

Reply:
Oh perfect, thanks! In the future, after coming up with the main group branch, we should iterate on the code collaboratively :)